test/integration/targets/ec2_vpc_nacl/tasks/main.yml

```diff
@@ -0,0 +1,143 @@
+---
+- block:
+
+    # ============================================================
+
+    - name: test without any parameters
+      ec2_vpc_nacl:
+      register: result
+      ignore_errors: yes
+
+    - name: assert required parameters
+      assert:
+        that:
+          - result.failed
+          - "result.msg == 'one of the following is required: name, nacl_id'"
+
+    # ============================================================
+
+    - name: set connection information for subsequent tasks
+      set_fact:
+        aws_connection_info: &aws_connection_info
```

Comment:
Rather than the eternal C&P / YAML hack this is better done with
```
module_defaults:
  group/aws:
    aws_access_key: "{{ aws_access_key | default(omit) }}"
    aws_secret_key: "{{ aws_secret_key | default(omit) }}"
    security_token: "{{ security_token | default(omit) }}"
    region: "{{ aws_region | default(omit) }}"
```

The common set_fact block doesn't work if you don't pass security_token, which isn't necessary for simply testing rigs.